### DISCUSSION TITLE
Valkey 7.2 or 8.0: Redis Timeseries and RediSearch

### DISCUSSION THREAD

**Discussion by wernermorgenstern** [valkey-io/valkey#1055] [Q&A]
I was wondering if anybody has been able to use the two modules (Timeseries and RediSearch from Redis) with Valkey.

Here is where I got so far:
Valkey 7.2 Clustered
Able to load both modules, however, unable to execute TS.ADD or FT.CREATE

Valkey 8.0 Clustered
Able to load Timeseries, unable to execute TS.ADD
Unable to load RediSearch (Valkey Node crashes - I submitted a crash report)

Versions for the modules I tried:
Search: 2.10
Timeseries: 1.12

**Comment by PingXie** [comment]
>Here is where I got so far:
Valkey 7.2 Clustered
Able to load both modules, however, unable to execute TS.ADD or FT.CREATE

Can you file a bug with detailed information for this issue? 

>Valkey 8.0 Clustered
Able to load Timeseries, unable to execute TS.ADD
Unable to load RediSearch (Valkey Node crashes - I submitted a crash report)

The crash is understood. The failure to execute TS.ADD might be related to 7.2 but hard to say without details.
